app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import datetime
+import time
+
+
+def every_second_file():
+    while True:
+        date_now = datetime.datetime.now()
+        full_date = f"app-{date_now.hour}" \
+                    f"_{date_now.minute}" \
+                    f"_{date_now.second}.log"
+
+        with open(full_date, 'w') as file:
+            file.write(str(date_now))
+
+        print(str(date_now) + ' ' + full_date)
```

Comment:
You can use in the print just variables date_now and full_date with coma between (without "str", "+" and ' '.)

Reply:
Thank you, I got it, what you mean